src/main/java/org/openstreetmap/atlas/geography/atlas/items/Node.java

```diff
@@ -53,6 +55,23 @@ public ItemType getType()
      */
     public abstract SortedSet<Edge> outEdges();
 
+    @Override
+    public String toDiffViewFriendlyString()
+    {
+        final Set<Relation> relations = this.relations();
```

Comment:
The same parent relation code is duplicated for every entity. Wondering if we can move this up to the `AtlasEntity` class and call `parentRelationsAsDiffViewFriendlyStrings`() in the implementations.

Reply:
Ah this is a good idea.